example/tsconfig.spec.json

```diff
@@ -1,15 +1,12 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "allowJs": true,
+    "emitDecoratorMetadata": true,
     "outDir": "./out-tsc/spec",
     "module": "commonjs",
-    "types": ["jest", "node"],
-    "emitDecoratorMetadata": true,
-    "allowJs": true
+    "types": ["jest", "node"]
```

Comment:
I would prefer to have the ordering as it is in `tsconfig.spec.json` of a newly generated with all additional props coming afterwards, this way it is easier for new users to figure out the differences.

Reply:
done